src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -50,8 +50,6 @@ trait RedisTrait
      */
     private function init($redisClient, string $namespace, int $defaultLifetime, ?MarshallerInterface $marshaller)
     {
-        parent::__construct($namespace, $defaultLifetime);
```

Comment:
Add this to mention this among BC breaks

Reply:
not needed: we don't document changes to internal code